Applications/meson.build

```diff
@@ -9,6 +9,7 @@ if enable_ccapi
 endif
 subdir('VGG/jni')
 subdir('Resnet/jni')
+subdir('YOLO')
```

Comment:
may be add for the android?

Reply:
I added jni folder and related files. thanks!